src/app/zap-templates/zcl/data-model/chip/chime-cluster.xml

```diff
@@ -0,0 +1,57 @@
+<?xml version="1.0"?>
+<!--
+Copyright (c) 2024 Project CHIP Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+<configurator>
+  <domain name="CHIP"/>
+
+  <struct name="ChimeSoundStruct" apiMaturity="provisional">
+    <cluster code="0x0556"/>
+    <item name="ChimeId" type="int8u"/>
+    <item name="Name" type="char_string" max="48"/>
+  </struct>
+
+  <cluster apiMaturity="provisional">
```

Comment:
Where is the spec for this?  I don't see it in the Matter appclusters spec on "master"....

It's impossible to review this XML without the spec.

Reply:
it's in the Cameras branch